alf/networks/critic_networks_test.py

```diff
@@ -118,13 +118,15 @@ def test_make_parallel(self):
 
         def _train(pnet, name):
             t0 = time.time()
-            optimizer = alf.optimizers.AdamTF(list(pnet.parameters()), lr=1e-4)
+            optimizer = alf.optimizers.AdamTF(lr=1e-4)
+            optimizer.add_param_group({'params': list(pnet.parameters())})
```

Comment:
what's the reason for this change?

Reply:
Before the init is wrong because the API is no longer like that.